python/graphstorm/run/gs_gen_embedding.py

```diff
@@ -0,0 +1,50 @@
+"""
```

Comment:
can we call it `gs_gen_node_embedding.py`?

Reply:
Yep. I have changed the naming and will leave the folder name still gsgnn_emb, where we can put gsgnn_node_emb.py here, in the future we can put gsgnn_edge_emb.py and gsgnn_graph_emb.py here as well.